### ISSUE TITLE
[FEAT] Work with macro folders

### ISSUE THREAD

**Issue by QuantumFunks** [arcanistzed/sidebar-macros#4]
Is there a way to make this work with the macro folders mod?

Its awesome to have macros available on the sidebar but i have well over 300 macros in use and scrolling through them is a nightmare. Having them all listed without folders or organisation makes this mod unusable unfortuantely.

**Comment by QuantumFunks** [comment]
Just saw on the discord that this is being implemented in V9.

**Comment by arcanistzed** [comment]
Yeah, it already works in v9 without any effort from me, so I decided not to spend time on this. See #2

**Comment by QuantumFunks** [comment]
Wait so it should currently be working? Im not getting folders. Are you saying it will work with V9 of foundry when its released?

**Comment by arcanistzed** [comment]
I'm saying it already works on the development version of v9.
